@navikt/core/tokens/darkside/util.ts

```diff
@@ -0,0 +1,51 @@
+import _ from "lodash";
+
+export const colorThemes = ["light", "dark"] as const;
+export type ColorTheme = (typeof colorThemes)[number];
+
+export const globalColorRoles = [
+  "neutral",
+  "accent",
+  "success",
+  "warning",
+  "danger",
+  "info",
+  "brandOne",
+  "brandTwo",
+  "brandThree",
+  "dataOne",
+  "dataTwo",
+  "dataThree",
+] as const;
+
+export type GlobalColorRoles = (typeof globalColorRoles)[number];
+
+export const globalColorScales = [
+  "100",
+  "200",
+  "300",
+  "400",
+  "500",
+  "600",
+  "700",
+  "800",
+  "900",
+  "1000",
+] as const;
+
+export type GlobalColorScale = (typeof globalColorScales)[number];
+
+export type GlobalColorVariable = Record<
+  GlobalColorScale,
+  { value: string; type: "global-color"; comment?: string; group: string }
+>;
+
+export const tokensWithPrefix = (
+  input: Record<string, any>,
+): Record<"a", any> => {
+  return { a: input };
+};
+
+export const mergeConfigs = (configs: any[]): Record<string, any> => {
```

Comment:
Can we create a type instead of `any`? Here is a suggestion/start (you might want to find a better name)

```suggestion
type TokenObj = {
  [type: string]: {
    [scale: string]: {
      value: string;
      type: string;  // Maybe we can list the possible types
      comment?: string;
      group?: string;
    };
  };
};

export const mergeConfigs = (configs: TokenObj[]): TokenObj => {
```

Reply:
Configurations can in theory have different depths, so these give the same output. 
```
a: {
 bg: { 
    [role]: {
       default: {
           ...
        }
    }
  }
}

a: {
 bg: { 
    [`${role}-default`]: {
         ...
    }
  }
}
```

I can try to create a type that accepts different depths tho.

Reply:
We can fix it later if it's cumbersome

Reply:
Fixed with this snippet
```ts
type TokenTypes = "color" | "global-color" | "global-radius" | "global-spacing";

export type StyleDictionaryToken<T extends TokenTypes> = {
  value: string;
  type: T;
  group?: string;
  comment?: string;
};

export type StyleDictionaryTokenConfig<T extends TokenTypes> = {
  [key: string]: StyleDictionaryToken<T> | StyleDictionaryTokenConfig<T>;
};
```

Reply:
Nice! But did you forget to update the types on `mergeConfigs`?

Reply:
Whoops, fixed now